vcfa/resource_vcfa_org_regional_networking_vpc_qos.go

```diff
@@ -0,0 +1,285 @@
+package vcfa
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/go-vcloud-director/v3/govcd"
+	"github.com/vmware/go-vcloud-director/v3/types/v56"
+)
+
+const labelVcfaOrgRegionalNetworkingVpcQos = "Regional Networking VPC Connectivity Profile QoS"
+
+func resourceVcfaOrgRegionalNetworkingVpcQos() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcfaOrgRegionalNetworkingVpcQosCreateUpdate,
+		ReadContext:   resourceVcfaOrgRegionalNetworkingVpcQosRead,
+		UpdateContext: resourceVcfaOrgRegionalNetworkingVpcQosCreateUpdate,
+		DeleteContext: resourceVcfaOrgRegionalNetworkingVpcQosDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcfaOrgRegionalNetworkingVpcQosImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"org_regional_networking_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: fmt.Sprintf("ID of %s", labelVcfaRegionalNetworkingSetting),
+			},
+			"edge_cluster_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: fmt.Sprintf("ID of parent %s", labelVcfaEdgeCluster),
+			},
+			"ingress_committed_bandwidth_mbps": {
+				Type:        schema.TypeString, // string + validation due to usual problem of differentiation between 0 and empty value for TypeInt
+				Optional:    true,
+				Computed:    true,
+				Description: fmt.Sprintf("Ingress committed bandwidth in Mbps for %s", labelVcfaEdgeCluster),
```

Comment:
This one and below, should it be
```suggestion
				Description: fmt.Sprintf("Ingress committed bandwidth in Mbps for %s", labelVcfaRegionalNetworkingSetting),
```

unless you are meaning here that it overrides the Edge Cluster? Then maybe we should change to `labelVcfaEdgeCluster` those in the data source

Reply:
No, you are right. I copied this chunk from Edge Cluster (because it is the same structure) and left it out while I made it correct for data source.
To be precise it is `labelVcfaOrgRegionalNetworkingVpcQos`, not `labelVcfaRegionalNetworkingSetting`
Fixed.